### ISSUE TITLE
Image resizeMode not working

### ISSUE THREAD

**Issue by jbcullis** [microsoft/react-native-windows#9305]
### Problem Description

When adding resizeMode={"contain"} the image fills the container.

### Steps To Reproduce

Create an image element, add resizeMode={'contain'}

### Expected Results

Image should be contained within the container.

### CLI version

npx react-native --version

### Environment

```markdown
OS: Windows 10 10.0.22000
    CPU: (16) x64 Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz
    Memory: 18.16 GB / 31.73 GB
  Binaries:
    Node: 16.13.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.17 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 8.1.3 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  SDKs:
    Android SDK: Not Found
    Windows SDK:
      AllowDevelopmentWithoutDevLicense: Enabled
      AllowAllTrustedApps: Enabled
      Versions: 10.0.18362.0, 10.0.19041.0, 10.0.22000.0
  IDEs:
    Android Studio: Not Found
    Visual Studio: 17.0.31912.275 (Visual Studio Community 2022)
  Languages:
    Java: Not Found
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.2 => 17.0.2
    react-native: 0.66.0 => 0.66.0
    react-native-windows: 0.66.5 => 0.66.5
  npmGlobalPackages:
    *react-native*: Not Found
```

### Target Platform Version

10.0.19041

### Target Device(s)

Desktop

### Build Configuration

Debug

**Comment by rozele** [comment]
This is fixed in v0.67 by f1910dbc63b8bdd00632c399629a2b6d6af96e03, @chrisglein perhaps #8854 should be cherry-picked to v0.66.

**Comment by jbcullis** [comment]
Ahh, sorry, I should have checked the closed issues before posting.
Implemented and resolved...thanks!
